### ISSUE TITLE
no netto prices using paypal as payment method while using swag business essential plugin

### ISSUE THREAD

**Issue by kuschneider** [PAYONE-GmbH/shopware-5#226]
Setting up an b2b shop with payone using paypal as payment method doesn't work correctly.

s_core_customergroup setup a group with "tax"=0 (deactivates showing brutto prices in storefront)

After placing an order the columns "net" and "taxfree" of the s_order table are 0 | 0, they should be 1 | 0, because of it's a customer group which shows netto prices in storefront but needs to display taxes at invoicing etc.

The wrong net parameter results into wrong invoice pdfs! The table header of the position prices is named "brutto" instead of "netto". Also the complete price calculation is wrong in the invoice pdf.

Using payone credit card (for example visa) works correctly, it pastes 1 | 0 in the s_order table and generates correct invoice pdfs.

**Comment by fjbender** [comment]
I couldn't reproduce the problem, using the same configuration in SW 5.4.6 with Plugin 3.10.0:

```
mysql> SELECT o.id, o.ordernumber, o.paymentID, o.net, o.taxfree, p.name FROM s_order o JOIN s_core_paymentmeans p ON o.paymentID = p.id WHERE o.ordernumber IN (20004, 20005);
+----+-------------+-----------+-----+---------+-----------------------------+
| id | ordernumber | paymentID | net | taxfree | name                        |
+----+-------------+-----------+-----+---------+-----------------------------+
| 14 | 20004       |         7 |   1 |       0 | mopt_payone__cc_visa        |
| 17 | 20005       |        22 |   1 |       0 | mopt_payone__ewallet_paypal |
+----+-------------+-----------+-----+---------+-----------------------------+
2 rows in set (0,00 sec)
```

Is there anything else that might be different in your setup?

**Comment by kuschneider** [comment]
The only difference I can see at this point is that we are using shopware 5.3.5. Could you try it again with this version? Or do you have any hints at which point we can check our configuration?

**Comment by kuschneider** [comment]
`

id | ordernumber | paymentID | net | taxfree | name
-- | -- | -- | -- | -- | --
363 | 30093 | 21 | 0 | 0 | mopt_payone__ewallet_paypal
360 | 0 | 21 | 1 | 0 | mopt_payone__ewallet_paypal

`
One thing we can see is, that before order is saved the net column is okay, after saveing the order it gets changed to zero :-/

**Comment by fjbender** [comment]
@hreinberger Can you try and reproduce this with the Swag Business Essentials Plugin?

**Comment by kuschneider** [comment]
@fjbender @hreinberger 
Did you find any solution for this problem?

**Comment by hreinberger** [comment]
Hi @kuschneider 
not yet but we're on it!

**Comment by hreinberger** [comment]
OK so we were finally able to reliably reproduce this issue and attemped a fix with #263 - can you try that out? We also found a kinda related bug with Paypal Express which should be fixed in #262
